### DISCUSSION TITLE
Reimagining CEABiGR (Comparative Epigenomic Analyses across Bivalve Genome Resources)

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1139] [Ideas]
We should soon get about 10k to support CEABiGR, the next generation of FROGER (bigger 😃).

I would love to brainstorm some ideas on ways to best leverage...

**Comment by sr320** [comment]
idea 1 - purchase Hyak node for general use

**Comment by sr320** [comment]
Host meeting at FHL (in person)

**Comment by sr320** [comment]
Somehow leverage other efforts such as @ksil91's Marine Genomics group..

**Reply by ksil91** [reply]
Ooo cool! Is the idea to develop resources for comparing epigenomic analyses between different bivalve species? And/or integrating different epigenomic datasets for the same species?

**Reply by sr320** [reply]
possibilities are endless... just needs to have some hint towards aquaculture.

**Comment by sr320** [comment]
Develop online data portal for multiple species and analysis containers (nextflow) - might involve compensating effort

**Reply by yaaminiv** [reply]
I like the idea of an online portal (Shiny app?) for multiple species/something where you can easily sort and pull methylation data by sequencing type, species, etc.

**Reply by sr320** [reply]
As I am learning more about what @ksil91 is doing - this likely falls inline with what @ksil91 working group is doing..

**Comment by yaaminiv** [comment]
1. meta analysis of all the methylation data we have, because we have a lot. could complement creation of an online portal

**Reply by sr320** [reply]
certainly this is low hanging fruit.

**Comment by sr320** [comment]
via @yaaminiv - (_separating ideas_) - can we use some of the 10k funds to create an internship (similar to what the MarineOmics group is doing),

**Reply by sr320** [reply]
I like this.. at what level were you thinking - undergraduate - grad?

**Reply by yaaminiv** [reply]
Undergraduate or early-stage graduate? Just thinking about a student who doesn't have much experience that could learn a lot from this kind of opportunity

**Comment by sr320** [comment]
via @yaaminiv - (_separating ideas_) - create outreach materials?
